lib/matplotlib/image.py

```diff
@@ -353,7 +354,7 @@ def draw(self, renderer, *args, **kwargs):
         if not self.get_visible():
             return
         if (self.axes.get_xscale() != 'linear' or
-            self.axes.get_yscale() != 'linear'):
+                self.axes.get_yscale() != 'linear'):
```

Comment:
OK, I see the logic for these now; the extra indentation is specifically for conditionals, so that the continuation of the condition will be clearly distinguished from the code block that follows.